### ISSUE TITLE
Free mini-course from SitePoint

### ISSUE THREAD

**Issue by newswim** [dwyl/learn-elm#76]
https://www.sitepoint.com/premium/courses/elm-a-beginners-guide-to-elm-and-data-2940

Runtime: approx. 2 hours

**Comment by iteles** [comment]
@newswim Would you like to PR this into the resources section at the bottom of the readme so it's more accessible? 😊

**Comment by newswim** [comment]
Suuure!

I've just been posting a lot of links as issues, rather than adding to the readme--😅.

There are a couple things I'd like to add while I work through `elm-workshop`, including adding transcripts and submitting a PR for the piece on *Language Features*. Expect more PR's soon-ish.

**Comment by nelsonic** [comment]
@newswim just to say that it _might_ not be worth it to add transcripts to `elm-workshop` 
because [Richard Feldman](https://twitter.com/nelsonic/status/839592853324066816) will be updating the content at some point in the _near_ future when `elm-lang@0.19` is released ... (_i.e. unless he repeats his content word-for-word in the training, there will be a lot to change!_)
We discussed it at the Elm Meetup: https://twitter.com/nelsonic/status/839592853324066816 😉

**Comment by newswim** [comment]
Hey @nelsonic, so cool that DWYL showed up to Mr. Feldman's talk!! (also see my initial reply to that tweet 😉)

Thanks for that info on the course update. The earlier sections seem to mainly be general info, so I imagine that material not changing _too_ drastically. I've started on [section 5](https://github.com/newswim/elm-workshop/blob/master/part5/Transcript.md) and [section 4](https://github.com/newswim/elm-workshop/blob/master/part4/Transcript.md), which will hopefully be of some use, but even if not.. it's a benefit to me as I learn. If that's not something that should go in this repo, that's totally cool!

**Comment by nelsonic** [comment]
@newswim yeah, agree that the content in the first videos should not change. 🤔 
if you have time to transcribe go for it! 👍
